src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/SplitFileService.cs

```diff
@@ -80,23 +81,36 @@ IImportStatusService importStatusService
             // If no batching was necessary, simply add a message to process the lone data file
             if (!batchFilesForDataFile.Any())
             {
-                collector.Add(message);
+                var observationsFilePath = 
+                    AdminReleaseDirectoryPath(message.Release.Id, ReleaseFileTypes.Data) + message.DataFileName;
```

Comment:
You can use `AdminReleasePath(message.Release.Id, ReleaseFileTypes.Data, message.DataFileName);` here.

Reply:
Done, thanks